paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1251,8 +1251,14 @@ public class CoreOptions implements Serializable {
             key("record-level.time-field")
                     .stringType()
                     .noDefaultValue()
+                    .withDescription("Time field for record level expire.");
+
+    public static final ConfigOption<TimeFieldType> RECORD_LEVEL_TIME_FIELD_TYPE =
+            key("record-level.time-field.type")
```

Comment:
record-level.field-type

Reply:
It is OK

Reply:
Fixed in https://github.com/apache/paimon/pull/3992/commits/20a186ed4c7abbf08f9b89ae33f74a5075d068dd.